build.gradle.kts

```diff
@@ -384,10 +385,7 @@ tasks.register<Test>("debug-community-docker") {
 }
 
 tasks.register<Test>("in-container-aurora") {
-    filter.includeTestsMatching("testsuite.integration.container.AuroraMysqlIntegrationTest")
-    filter.includeTestsMatching("testsuite.integration.container.AuroraMysqlFailoverIntegrationTest")
-    filter.includeTestsMatching("testsuite.integration.container.AuroraMysqlHardFailureIntegrationTest")
-    filter.includeTestsMatching("testsuite.integration.container.ReplicationFailoverIntegrationTest")
+    filter.includeTestsMatching("testsuite.integration.container.IntegrationTestSuite")
```

Comment:
If we want to control order of execution for integration tests then I'd recommend add a comment here and explain how to add more test classes.